lib/path.js

```diff
@@ -340,11 +340,20 @@ win32.basename = function(path, ext) {
   if (ext !== undefined && typeof ext !== 'string')
     throw new TypeError('"ext" argument must be a string');
 
-  var f = win32SplitPath(path)[2];
-  // TODO: make this comparison case-insensitive on windows?
-  if (ext && f.substr(-1 * ext.length) === ext) {
```

Comment:
For readability, it could be: `if (ext && f.toLowerCase().endsWith(ext.toLowerCase())) {`